release-pipeline.yml

```diff
@@ -0,0 +1,1737 @@
+variables:
+  AZCOPY_GOLANG_VERSION: '1.21.8'
+  AZCOPY_GOLANG_VERSION_COVERAGE: '1.21.8'
+  AZCOPY_VERSION: '0.0.0'
+  MUTEX_URL: https://ciazcopyv10regular.blob.core.windows.net/distributedmutex/lock
+
+parameters:
+  - name: tag
+    displayName: 'New Release Tag'
+    type: string
+    default: 'azcopy-'
+
+  - name: test_mode
+    displayName: 'Test Mode'
+    type: boolean
+    default: false
+
+  - name: post_release
+    displayName: 'Post Release on Github'
+    type: boolean
+    default: false
+
+  - name: publish_artifacts
+    displayName: 'Publish Artifacts to Linux Repos'
+    type: boolean
+    default: false
+
+  - name: draft
+    displayName: 'Post as Draft Release'
+    type: boolean
+    default: false
+
+  - name: prerelease
+    displayName: 'Post as PreRelease'
+    type: boolean
+    default: false
+
+  - name: update_version
+    displayName: 'Update Version'
+    type: boolean
+    default: false
+
+# Do not trigger this pipeline automatically
+trigger: none
+pr: none
+
+stages:
+  - stage: BuildArtifacts
+    jobs:
+      - job: Set_1
+        timeoutInMinutes: 120
+        strategy:
+          matrix:
+            Ubuntu_amd64:
+              imageName: 'ubuntu-20.04'
+              container: 'test-cnt-ubn-20'
+              AgentName: "blobfuse-ubuntu20"
+        pool:
+          name: "blobfuse-ubuntu-pool"
+          demands:
+            - ImageOverride -equals $(agentName)
+
+        variables:
+          - name: root_dir
+            value: '$(System.DefaultWorkingDirectory)'
+          - name: work_dir
+            value: '$(System.DefaultWorkingDirectory)/azure-storage-azcopy'
+          - name: archives
+            value: $(root_dir)/archives
+          - name: linux_amd64
+            value: $(work_dir)/linux_amd64
+          - name: linux_se_amd64
+            value: $(work_dir)/linux_se_amd64
+
+        steps:
+          #TODO: remove this after merging to main
+          - script: |
+              wget https://raw.githubusercontent.com/dphulkar-msft/azure-storage-azcopy/az-pipelineTest/go_installer.sh -P $(work_dir)/
```

Comment:
Make sure to update this link after this PR is merged

Reply:
yes